hatch/files/setup.py

```diff
@@ -17,52 +23,54 @@
 with open('{readme_file}', 'r', encoding='utf-8') as f:
     readme = f.read()
 
-REQUIRES = []
-
-setup(
-    name='{package_name}',
-    version=version,
-    description='',
-    long_description=readme,
-    author='{name}',
-    author_email='{email}',
-    maintainer='{name}',
-    maintainer_email='{email}',
-    url='{package_url}',
-    license='{license}',
-
-    keywords=[
-        '',
-    ],
-
-    classifiers=[
+REQUIRES = {requires}
+
+kwargs = {{
+    'name': '{package_name}',
+    'version': version,
+    'description': '',
+    'long_description': readme,
+    'author': '{name}',
+    'author_email': '{email}',
+    'maintainer': '{name}',
+    'maintainer_email': '{email}',
+    'url': '{package_url}',
+    'license': '{license}',
+    'classifiers': [
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',{license_classifiers}
         'Natural Language :: English',
         'Operating System :: OS Independent',{pyversions}
-        'Programming Language :: Python :: Implementation :: CPython',{pypy}    ],
```

Comment:
We might need to keep it like this to get proper output from our `parse` test dependency. We'll see.